apps/cookbook/src/app/showcase/label-showcase/label-showcase.component.html

```diff
@@ -0,0 +1,27 @@
+<h2>Label Showcase</h2>
+
```

Comment:
We should add a small text about what the label component is, similar to the intro text of header, item, badge and so on.

I do not think one currently exists in the design docs, so we should create on with @henrikvoetmand